.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,17 @@
+name: Run Unit Tests
+on:
+  pull_request: {}
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install pytest
+          pip install .
+      - name: Run tests
+        run: pytest
```

Comment:
I'd be happy to replace this with a managed GHA, but I think I went this route because the existing Neon workflow is looking for some skill-specific things that don't apply